connector-runtime/connector-runtime-spring/src/main/java/io/camunda/connector/runtime/inbound/webhook/InboundWebhookRestController.java

```diff
@@ -128,9 +137,12 @@ private ResponseEntity<?> processWebhook(
         // when verification was skipped
         // Step 2: trigger and correlate
         var webhookResult = connectorHook.triggerWebhook(payload);
-        var ctxData = toWebhookTriggerResultContext(webhookResult);
+        // create documents if the connector is activable
+        var documentReferences = createDocuments(connector.context(), payload);
```

Comment:
How do we deal with errors during `createDocuments`? Will the exceptions just bubble up?

Reply:
@johnBgood and for now we would just swallow the exceptions?

Reply:
For future reference, it's been discussed and we decided to leave it as it is now. When the Document client returns a Result it will be used here for proper error handling.